geometric_kernels/spaces/homogeneous_spaces.py

```diff
@@ -148,15 +148,15 @@ def num_levels(self) -> int:
     @property
     def num_eigenfunctions(self) -> int:
         """Number of eigenfunctions, M"""
-        return self._num_eigenfunctions
+        return self._num_levels
```

Comment:
Cannot be true, levels are characters for the homogeneous spaces, there's a bunch of eigenfunctions per each level.

Reply:
Then why does `num_eigenfunctions_per_level` return a list of ones?

Reply:
This seems to follow the convention as in the rest of the code, that ties in the methods `__call__`, `num_eigenfunctions`, `num_levels`, `num_eigenfunctions_per_level`, and `get_repeated_eigenvalues`. Seems like we can safely make this un-implemented?

Reply:
Okay. It would be great if you could create this other PR soon, before we forget about this.